docs/en/observability/logs-stream.asciidoc

```diff
@@ -1,114 +1,128 @@
 [[logs-stream]]
 = Stream a log file
 
-In this guide, you'll learn how to take a log file from your host and send it to Elasticsearch using a standalone {agent}. You'll configure the {agent} and your data streams using the Custom Logs integration. From there, you'll learn how to query your logs and use the data streams you've set up to have more control and flexibility when filtering your log data. 
+In this guide, you'll learn how to send a log file to Elasticsearch using a standalone {agent}, configure the {agent} and your data streams using the `elastic-agent.yml` file, and query your logs using the data streams you've set up. 
 
 [discrete]
 [[logs-stream-prereq]]
-= Prerequisites
+== Prerequisites
 
 include::logs-metrics-get-started.asciidoc[tag=monitoring-prereqs]
 
 [discrete]
 [[logs-stream-install-config-agent]]
-= Install and configure the standalone {agent}
+== Install and configure the standalone {agent}
 
-Install and configure the standalone {agent} to send your log data to {es} by completing the following steps:
+Complete the following steps to install and configure the standalone {agent} and send your log data to {es}:
 
-. Download and extract the {agent} installation package.
-. Configure the {agent} using the Custom Logs integration.
-. Install and start the {agent}.
+. <<logs-stream-extract-agent, Download and extract the {agent} installation package.>>
+. <<logs-stream-install-agent, Install and start the {agent}.>>
+. <<logs-stream-agent-config, Configure the {agent}.>>
 
 [discrete]
 [[logs-stream-extract-agent]]
-== Step 1: Download and extract the {agent} installation package
+=== Step 1: Download and extract the {agent} installation package
 
 On your host, download and extract the installation package that corresponds with your system:
 
 include::{ingest-docs-root}/docs/en/ingest-management/tab-widgets/download-widget.asciidoc[]
 
+[discrete]
+[[logs-stream-install-agent]]
+=== Step 2: Install and start the {agent}
+After downloading and extracting the installation package, you're ready to install the {agent}. From the agent directory, run the the install command that corresponds with your system:
+
+NOTE: On macOS, Linux (tar package), and Windows, run the `install` command to
+install {agent} as a managed service and start the service. The DEB and RPM
+packages include a service unit for Linux systems with
+systemd, so just enable then start the service.
```

Comment:
I realize this was copied from the Fleet Guide, but this text is confusing to me.

> On macOS, Linux (tar package), and Windows, run the install command to install Elastic Agent as a managed service and start the service.

This sounds like you have to install and then start, but I think you only have to install, right?

> The DEB and RPM packages include a service unit for Linux systems with systemd, so just enable then start the service.

This sounds like you have to install and then start, which you do. Right?

What about this instead?

```suggestion
NOTE: On macOS, Linux (tar package), and Windows, run the `install` command to
install and start {agent} as a managed service. The DEB and RPM
packages include a service unit for Linux systems with
systemd. For these systems, you must enable and start the service.
```

If you do decide to update this snippet, please update the source file too: https://github.com/elastic/ingest-docs/blame/main/docs/en/ingest-management/elastic-agent/install-standalone-elastic-agent.asciidoc